parts/linux/cloud-init/nodecustomdata.yml

```diff
@@ -445,15 +445,6 @@ write_files:
   content: !!binary |
     {{GetVariableProperty "cloudInitData" "customSearchDomainsScript"}}
 
-{{- if IsKubeletConfigFileEnabled}}
-- path: /etc/systemd/system/kubelet.service.d/10-componentconfig.conf
-  permissions: "0600"
-  encoding: gzip
```

Comment:
Does `encoding: gzip` need to be present in cse_config too?

Reply:
no, that only applies to these files: https://github.com/Azure/AgentBaker/blob/master/pkg/agent/variables.go#L22-L52